project.clj

```diff
@@ -13,15 +13,15 @@
   :codox {:source-uri "http://github.com/metosin/jsonista/blob/master/{filepath}#L{line}"
           :output-path "doc"
           :metadata {:doc/format :markdown}}
-  :dependencies [[com.fasterxml.jackson.core/jackson-databind "2.9.4"]]
+  :dependencies [[com.fasterxml.jackson.core/jackson-databind "2.9.5"]
+                 [com.fasterxml.jackson.datatype/jackson-datatype-jsr310 "2.9.5"]]
   :profiles {:dev {:dependencies [[org.clojure/clojure "1.9.0"]
+                                  [com.fasterxml.jackson.datatype/jackson-datatype-joda "2.9.5"]
                                   [cheshire "5.8.0"]
-                                  [criterium "0.4.4"]]
-                   :global-vars {*warn-on-reflection* true}}
```

Comment:
`*warn-on-reflection*` was on for a purpose: for performance, we don't want to do any calls via reflection.

Reply:
Problem is that with Cursive, there are a lot of false(?) positives with that setting:

```
Reflection warning, /private/var/folders/39/j64jmzc96xdd40ghbvw72rk80000gn/T/form-init6952961206932838438.clj:1:180 - reference to field getFileName can't be resolved.
Reflection warning, /private/var/folders/39/j64jmzc96xdd40ghbvw72rk80000gn/T/form-init6952961206932838438.clj:1:219 - reference to field getLineNumber can't be resolved.
Reflection warning, /private/var/folders/39/j64jmzc96xdd40ghbvw72rk80000gn/T/form-init6952961206932838438.clj:1:261 - reference to field getClassName can't be resolved.
Reflection warning, /private/var/folders/39/j64jmzc96xdd40ghbvw72rk80000gn/T/form-init6952961206932838438.clj:1:431 - reference to field getStackTrace can't be resolved.
Reflection warning, /private/var/folders/39/j64jmzc96xdd40ghbvw72rk80000gn/T/form-init6952961206932838438.clj:1:705 - reference to field getClassName can't be resolved.
Reflection warning, /private/var/folders/39/j64jmzc96xdd40ghbvw72rk80000gn/T/form-init6952961206932838438.clj:1:692 - call to method startsWith can't be resolved (target class is unknown).
Reflection warning, /private/var/folders/39/j64jmzc96xdd40ghbvw72rk80000gn/T/form-init6952961206932838438.clj:1:765 - reference to field getStackTrace can't be resolved.
Reflection warning, /private/var/folders/39/j64jmzc96xdd40ghbvw72rk80000gn/T/form-init6952961206932838438.clj:1:2287 - reference to field getName can't be resolved.
Reflection warning, /private/var/folders/39/j64jmzc96xdd40ghbvw72rk80000gn/T/form-init6952961206932838438.clj:1:4562 - reference to field flush can't be resolved.
```

Reply:
That's weird.